website/app/styles/pages/components/copy-snippet.scss

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+
+// COMPONENTS > COPY/COPY-SNIPPET
+
+.doc-page-constrain-copy-snippet-width {
```

Comment:
can you use the existing pattern/naming conventions for page-specific styles?
https://github.com/hashicorp/design-system/blob/main/website/app/styles/pages/components/modal.scss#L8-L9